gallery/src/Application.tsx

```diff
@@ -4,92 +4,116 @@
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#delta" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#epsilon" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#Zeta" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#Etha" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+              Nav Item
+            </span>
+          </a>
+          <a href="#Theta" className="nx-sidebar-navigation__link nx-text-link">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <NxOverflowTooltip>
+              <span className="nx-sidebar-navigation__text nx-page-sidebar-operable__expanded-content">
+                Est sint sunt ea laborum minim officia excepteur nostrud deserunt proident officia.
+              </span>
+            </NxOverflowTooltip>
+          </a>
+        </NxNavigationSidebarLinks>
+        <NxNavigationSidebarContent className="custom-tree-content nx-page-sidebar-operable__expanded-content">
+          <NxTreeView onToggleCollapse={onToggleCollapse}
+                      isOpen={toggleCheck}
+                      triggerContent="TreeView Content">
+            <NxTreeViewChild>
+              <NxTreeView onToggleCollapse={onToggleCollapse2}
+                          isOpen={toggleCheck2}
+                          triggerContent="Tree View Child">
+                <NxTreeViewChild>Tree View Item Nested</NxTreeViewChild>
+                <NxTreeViewChild>Tree View Item Nested</NxTreeViewChild>
+              </NxTreeView>
+            </NxTreeViewChild>
+            <NxTreeViewChild>Tree View Item</NxTreeViewChild>
+            <NxTreeViewChild>Tree View Item</NxTreeViewChild>
+            <NxTreeViewChild>Tree View Item</NxTreeViewChild>
+          </NxTreeView>
+        </NxNavigationSidebarContent>
+      </NxStatefulNavigationSidebar>
+      <div className="nx-page-main">
```

Comment:
Minor: use `<main>`.

Reply:
> Minor: use `<main>`.

Done in 0715c63fc2d3b354b58a1956db68d34314d393f9